python/ray/services.py

```diff
@@ -1276,502 +1266,248 @@ def start_raylet_monitor(redis_address,
     """Helper method to start Ray processes.
 
     Args:
-        address_info (dict): A dictionary with address information for
-            processes that have already been started. If provided, address_info
-            will be modified to include processes that are newly started.
-        object_manager_ports (list): A list of the ports to use for the object
-            managers. There should be one per object manager being started on
-            this node (typically just one).
-        node_manager_ports (list): A list of the ports to use for the node
-            managers. There should be one per node manager being started on
-            this node (typically just one).
-        node_ip_address (str): The IP address of this node.
-        redis_port (int): The port that the primary Redis shard should listen
-            to. If None, then a random port will be chosen. If the key
-            "redis_address" is in address_info, then this argument will be
-            ignored.
-        redis_shard_ports: A list of the ports to use for the non-primary Redis
-            shards.
-        num_workers (int): The number of workers to start.
-        num_local_schedulers (int): The total number of local schedulers
-            required. This is also the total number of object stores required.
-            This method will start new instances of local schedulers and object
-            stores until there are num_local_schedulers existing instances of
-            each, including ones already registered with the given
-            address_info.
-        object_store_memory: The amount of memory (in bytes) to start the
-            object store with.
-        redis_max_memory: The max amount of memory (in bytes) to allow redis
-            to use, or None for no limit. Once the limit is exceeded, redis
-            will start LRU eviction of entries. This only applies to the
-            sharded redis tables (task and object tables).
-        collect_profiling_data: Whether to collect profiling data. Note that
-            profiling data cannot be LRU evicted, so if you set
-            redis_max_memory then profiling will also be disabled to prevent
-            it from consuming all available redis memory.
-        num_redis_shards: The number of Redis shards to start in addition to
-            the primary Redis shard.
-        redis_max_clients: If provided, attempt to configure Redis with this
-            maxclients number.
-        redis_password (str): Prevents external clients without the password
-            from connecting to Redis if provided.
-        worker_path (str): The path of the source code that will be run by the
-            worker.
+        ray_params (ray.params.RayParams): The RayParams instance. The
+            following parameters will be set to default values if it's None:
+            node_ip_address("127.0.0.1"), num_local_schedulers(1),
+            include_log_monitor(True), worker_path(path of default_worker.py)
```

Comment:
Previously, `include_log_monitor`'s default value was `False`. But now it's `True`.

Reply:
Good catch...